go/billing/settings.py

```diff
@@ -43,7 +43,7 @@
 STATEMENT_CONTACT_DETAILS = getattr(
     settings, 'BILLING_STATEMENT_CONTACT_DETAILS', {
```

Comment:
I realised that for consistency this should be ``GO_STATEMENT_CONTACT_DETAILS``? I also wonder about having our own email addresses and web address set in the default settings? Maybe we should have dummy settings here and only have these values in our own production settings?